dipy/sims/voxel.py

```diff
@@ -288,7 +299,265 @@ def multi_tensor(gtab, mevals, S0=100, angles=[(0, 0), (90, 0)],
+    for i in range(len(fractions)):
+        D=D+fractions[i]*mD[i]    
+    dt=np.array([D[0][0],D[1][1],D[2][2],D[0][1],D[0][2],D[1][2]])
+    
+    # compute KT
+    kt=np.zeros((15))
+    kt[0] = compute_Wijkl(mD, fractions, 0,0,0,0, D)
+    kt[1] = compute_Wijkl(mD, fractions, 1,1,1,1, D)
+    kt[2] = compute_Wijkl(mD, fractions, 2,2,2,2, D)
+    kt[3] = compute_Wijkl(mD, fractions, 0,0,0,1, D)
+    kt[4] = compute_Wijkl(mD, fractions, 0,0,0,2, D)
+    kt[5] = compute_Wijkl(mD, fractions, 0,1,1,1, D)
+    kt[6] = compute_Wijkl(mD, fractions, 1,1,1,2, D)
+    kt[7] = compute_Wijkl(mD, fractions, 0,2,2,2, D)
+    kt[8] = compute_Wijkl(mD, fractions, 1,2,2,2, D)
+    kt[9] = compute_Wijkl(mD, fractions, 0,0,1,1, D)
+    kt[10] = compute_Wijkl(mD, fractions, 0,0,2,2, D)
+    kt[11] = compute_Wijkl(mD, fractions, 1,1,2,2, D)
+    kt[12] = compute_Wijkl(mD, fractions, 0,0,1,2, D)
+    kt[13] = compute_Wijkl(mD, fractions, 0,1,1,2, D)
+    kt[14]=  compute_Wijkl(mD, fractions, 0,1,2,2, D)
+    
+    # compute S based on the DT and KT
+    S=single_diffkurt_tensors(gtab, S0, dt, kt, snr)
+
+    return dt, kt, S
+    
+def compute_Wijkl(Dc,frac,i,j,k,el,DT=None):
+    r""" # function that computes the elements of KT by especifying their 
+    four indexes i, j, k and el. Each index can be equal to x, y or z which are 
+    codified respectively by 0, 1 and 2. Simulations are based on 
+    multicompartment models. Implementation of this function is based on 
+    equation 8 of:
+    
+    [1] R. Neto Henriques et al., "Exploring the 3D geometriy of the diffusion 
+        kurtosis tensor - Impact on the development of robust tractography 
+        procedures and novel biomarkers", NeuroImage (2015) 111, 85-99.
+    
+    Parameters
+    -----------
+    Dc : (6,) ndarray
+        Elements of the diffusion tensor for each individual compartment on the
+        multicompartmental model. Assumes the order Dxx, Dyy, Dzz, Dxy, Dxz, 
+        Dyz.
+    frac : float
+        Percentage of the contribution of each tensor. The sum of fractions
+        should be equal to 100%.
+    i : int
+        first element index (0 for x, 1 for y, 2 for z)
+    j : int
+        Second element index (0 for x, 1 for y, 2 for z)
+    k : int
+        Third element index (0 for x, 1 for y, 2 for z)
+    el: int
+        Fourth elements index (0 for x, 1 for y, 2 for z)
+    DT : (6,) ndarray
+        Elements of the global diffusion tensor.
+
+    Returns
+    --------
+    wijkl : Kurtosis tensor element
+    
+    """
+    if DT is None:
+        DT=np.zeros((3,3))
+        for i in range(len(frac)):
+            DT=DT+frac[i]*Dc[i]
+
+    wijkl=0
+    MD=(DT[0][0]+DT[1][1]+DT[2][2])/3
+    for f in range(len(frac)):
+        wijkl=wijkl+frac[f]*(Dc[f][i][j]*Dc[f][k][el]+Dc[f][i][k]*Dc[f][j][el]+
+                             Dc[f][i][el]*Dc[f][j][k])
+    wijkl=(wijkl-
+           DT[i][j]*DT[k][el]-DT[i][k]*DT[j][el]-DT[i][el]*DT[j][k])/(MD**2)
+    
+    return wijkl
```

Comment:
PEP8 in this entire function: space around operators and around `=`. Make sure to break lines into 80 characters when you add those spaces.